monitoring/deployment_manager/README.md

```diff
@@ -0,0 +1,37 @@
+## Purpose
+
+deployment_manager is an automation tool that will be capable of performing most of the steps a human would otherwise have to manually execute to:
+
+* Deploy a DSS instance, both [locally](../../build/dev/standalone_instance.md) and [in production](../../build/README.md)
+* [Troubleshoot a DSS instance](../../build/README.md)
+* [Form or maintain a DSS pool](../../build/pooling.md)
+* [Migrate database schemas](../../build/README.md#upgrading-database-schemas)
+* [Upgrade CockroachDB nodes](https://www.cockroachlabs.com/docs/stable/upgrade-cockroach-version.html)
+* Deploy a set of diagnostic tools ([tracer](../tracer), [uss_qualifier](../uss_qualifier), [mock_uss](../mock_uss))
+
+deployment_manager actions all accept a declarative definition of the InterUSS deployment desired.  To make changes to a deployment, the administrator should edit the development definition and then execute the desired action.
+
+## Usage
+
+To use deployment_manager, first define an InterUSS deployment in a JSON file according to the [DeploymentSpec](deployments.py) schema.  From a context with kubectl already configured to work with the target Kubernetes cluster, run [deployment_manager.py](deployment_manager.py) according to its help (`python3 deployment_manager.py --help`).
```

Comment:
Could you please add the few steps required to install dependencies and start the python3 script ?
I tried without success the following commands:
```
$ pip3 install -r requirements.txt
$ python3 deployment_manager.py --help
Traceback (most recent call last):
  File "[REDACTED]/dss/monitoring/deployment_manager/deployment_manager.py", line 10, in <module>
    from monitoring.monitorlib.typing import ImplicitDict
ModuleNotFoundError: No module named 'monitoring'
```

Reply:
Working steps:
```
cd monitoring/deployment_manager
export PYTHONPATH=$(pwd)/../..
pip3 install -r requirements.txt
pip3 install -r ../monitorlib/requirements.txt
python3 deployment_manager.py --help
```

Reply:
Done